hail/Makefile

```diff
@@ -1,4 +1,12 @@
-.PHONY: shadowJar
+.PHONY: shadowJar jars clean
 
 shadowJar:
 	./gradlew shadowJar
+
+jars:
+	make -C src/main/c
+	mvn compile test-compile jar:jar jar:test-jar assembly:single
```

Comment:
Doesn't assembly:single build both the jar and test-jar already? Why do we need the two jar steps?

Reply:
No, I found if I didn't do that then the test jar with dependencies didn't have any of the Hail test classes in it.  Of course, I really want to be able to say `mvn assembly:single` only and have it build the minimum dependencies, but I don't know how to do that without more research.